### ISSUE TITLE
adding support for linux-ppc64le  in CI 

### ISSUE THREAD

**Issue by adilhusain-s** [kubeflow/mpi-operator#487]
Hi there, 
I want to add support for linux-ppc64le in CI (github action) to release binaries/docker images.
the roadmap.md file mention that CICD needs to be implemented to release the docker image.
is this roadmap.md file UpToDate?
if yes, then what github action do we want to use to implement the release process?

I suggest using the goreleaser [github action](https://github.com/marketplace/actions/goreleaser-action).
I also want to understand how we're currently releasing the docker image for mpi-operator because I couldn't find any github workflow that does this. 
 
let's discuss this in the comments.

**Comment by terrytangyuan** [comment]
I am unsure if we should invest more time in the CI of this repo since new development will happen in the training-operator repo.

**Comment by adilhusain-s** [comment]
@terrytangyuan 
are we planning to archive this repo and move the development of mpi-operator to training-operator repo?
just want to understand what's the plan.

**Comment by terrytangyuan** [comment]
v1 has been migrated but v2 is still under discussion.

**Comment by alculquicondor** [comment]
https://github.com/kubeflow/training-operator/issues/1479

tl;dr there is some desire to do it, but also some desire to keep the repos separate so that it's simpler for users to deploy.

I think it's worth adding the CI for this repo.

**Comment by terrytangyuan** [comment]
Feel free to submit PRs if you'd like to help set this up.

**Comment by lehrig** [comment]
@adilhusain-s, we already enabled ppc64le CI for the training operator in https://github.com/kubeflow/training-operator/pull/1692, right?

@terrytangyuan that will mean that if https://github.com/kubeflow/training-operator/issues/1479 is fixed, it should also maintain ppc64le support, right?

I think we should still submit PRs here for these reasons:
- keeping separate repos as @alculquicondor mentioned
- maintaining consistency between the two repos
- making it easier to work on https://github.com/kubeflow/training-operator/issues/1479

(note: please find the ppc64le umbrella issue here: https://github.com/kubeflow/kubeflow/issues/6684)

**Comment by terrytangyuan** [comment]
> @terrytangyuan that will mean that if kubeflow/training-operator#1479 is fixed, it should also maintain ppc64le support, right?

Yes

**Comment by terrytangyuan** [comment]
This seems working now. Thank you! We also need to push latest tag. Would you like to send a PR as well?

**Comment by adilhusain-s** [comment]
@terrytangyuan 
the current workflow  uses latest git tag  as latest docker tag.
could you  specified the requirements?

[docker meta action](https://github.com/docker/metadata-action)
please see the second table in above link to understand this.

**Comment by terrytangyuan** [comment]
Please ignore my comment. Apparently latest tags have different meanings in different projects. :-)
